src/Rector/AbstractRector.php

```diff
@@ -132,7 +129,6 @@ public function autowire(
         PhpVersionProvider $phpVersionProvider,
         ExclusionManager $exclusionManager,
         StaticTypeMapper $staticTypeMapper,
-        ParameterProvider $parameterProvider,
```

Comment:
it need to be kept to avoid issue https://github.com/rectorphp/rector/issues/7117 happen again.

Reply:
How exactly do you use the parameter provider?

Reply:
it used in laminas-servicemanager-migration to replace use interop statement when auto import enabled: https://github.com/laminas/laminas-servicemanager-migration/blob/93cfd29ea5cf09fc634db9bf63f98ab377da0560/src/Rector/Class_/ImplementsFactoryInterfaceToPsrFactoryRector.php#L161-L188